Bio/AlignIO/StockholmIO.py

```diff
@@ -454,7 +454,16 @@ def __next__(self):
                 elif line[:5] == '#=GS ':
                     # Generic per-Sequence annotation, free text
                     # Format: "#=GS <seqname> <feature> <free text>"
-                    seq_id, feature, text = line[5:].strip().split(None, 2)
```

Comment:
Taking the specification literally, the free text could be spaces? If so, does this cause problems elsewhere?

```
seq_id, feature, text = line[5:].split(None, 2)
```

Also, should we not make a similar change to the ``"#=GF `` case above?